diff --git a/arch/arm/boot/dts/qcom/sdm636-mtp_e7s.dts b/arch/arm/boot/dts/qcom/sdm636-mtp_e7s.dts
index 702b6c139955..3d1ba7883fde 100644
--- a/arch/arm/boot/dts/qcom/sdm636-mtp_e7s.dts
+++ b/arch/arm/boot/dts/qcom/sdm636-mtp_e7s.dts
@@ -22,10 +22,9 @@
 / {
 	model = "Qualcomm Technologies, Inc. SDM 636 PM660 + PM660L E7S";
 	compatible = "qcom,sdm636-mtp", "qcom,sdm636", "qcom,mtp";
-	qcom,board-id = <0x30008 0>;
+	qcom,board-id = <0x10008 0>;
 	qcom,pmic-id = <0x0001001b 0x0101011a 0x0 0x0>,
-			<0x0001001b 0x0201011a 0x0 0x0>,
-			<0x0001001b 0x0102001a 0x0 0x0>;
+			<0x0001001b 0x0201011a 0x0 0x0>;
 };
 
 &tavil_snd {
diff --git a/arch/arm/boot/dts/qcom/sdm636_e7s.dtsi b/arch/arm/boot/dts/qcom/sdm636_e7s.dtsi
index 62eccd585032..25643aa2826e 100644
--- a/arch/arm/boot/dts/qcom/sdm636_e7s.dtsi
+++ b/arch/arm/boot/dts/qcom/sdm636_e7s.dtsi
@@ -33,7 +33,6 @@
 &soc {
 	/delete-node/ qcom,turing@1a300000;
 
-	/delete-node/ qcom,msm-cdsp-loader;
 	/delete-node/ cti@7068000;
 	/delete-node/ turing_etm0;
 	funnel@6042000 {
